docs/source/configuration/telemetry/exporters/logging/stdout.mdx

```diff
@@ -335,6 +335,42 @@ Example output with a list of spans:
 }
 ```
 
+#### `span_attributes`
+
+The `telemetry.exporters.logging.stdout.format.json.span_attributes` option is useful if you don't want to display all spans attributes but only some of them. It takes an array of attribute name to include as a root logging attribute.
+This will search for the first attribute in the list of span attributes from the root span to the current one and attach it to the outmost json object for the log event.
+If you set the same attribute on different spans at different level, then the identical attribute from child spans will take precedence over the one found previously in parent spans.
+
+For example, if you have spans that contains `graphql.document` attribute and you only want to display this span attribute:
+
+```yaml title="router.yaml"
+telemetry:
+  exporters:
+     logging:
+       stdout:
+         enabled: true
+         format: 
+           json:
+             display_span_list: false
```

Comment:
Should document the valid or recommended setting of `display_span_list`. I assume it's either recommended or required to be false?

Reply:
It's because usually when using span_attributes feature you probably don't want to display all the span list. Nothing prevents you from doing it, it's still valid but pretty useless